test/functional/test_env.yaml

```diff
@@ -1,13 +1,25 @@
+# To run these tests with TLC (for Boulder office)
+# 1. You can start a TLC sessions with 3 compute hosts and provision 6 floating IP addresses on your TLC file
```

Comment:
I am fixing this under a different PR into our local gitlab that manages the TLC deployment.